Yahoo_Page.py

```diff
@@ -58,77 +67,94 @@ def click_search_button(self):
         self.driver.switch_to.window(current_tab)        
 
     #Drop Downs
 
     def hover_over_profile_menu(self):
         profile_menu_loc = (By.XPATH,'//label[@role="presentation"]')
-        wait(self.driver,15).until(EC.visibility_of_element_located(profile_menu_loc))
+        wait(self.driver,15).until(
+            EC.visibility_of_element_located(profile_menu_loc))
         profile_menu_element = self.driver.find_element(*profile_menu_loc)                     
-        ActionChains(self.driver).move_to_element(profile_menu_element).perform()
+        ActionChains(self.driver).move_to_element(
+            profile_menu_element).perform()
 
     def click_profile_menu_settings(self):
         setting_link_loc = (By.LINK_TEXT,'Settings')
-        wait(self.driver,15).until(EC.element_to_be_clickable(setting_link_loc))
+        wait(self.driver,15).until(
+            EC.element_to_be_clickable(setting_link_loc))
         self.driver.find_element(*setting_link_loc).click()
     
     def hover_over_originals_drop_down(self):
         originals_drop_down_loc = (By.XPATH,'//a[@title="Originals"]')
-        wait(self.driver,15).until(EC.visibility_of_element_located(originals_drop_down_loc))
+        wait(self.driver,15).until(
+            EC.visibility_of_element_located(originals_drop_down_loc))
         drop_down_element = self.driver.find_element(*originals_drop_down_loc)
         ActionChains(self.driver).move_to_element(drop_down_element).perform()
 
     #Tabs and Options
 
     def click_account_security_tab(self):       
         account_security_loc = (By.PARTIAL_LINK_TEXT,'Account Security')
-        wait(self.driver,15).until(EC.presence_of_element_located(account_security_loc))
+        wait(self.driver,15).until(
+            EC.presence_of_element_located(account_security_loc))
         self.driver.find_element(*account_security_loc).click()
 
     def click_random_option_from_originals_drop_down(self):
-        all_options_loc = (By.XPATH,'//a[@title="Originals"]/following-sibling::div//a')
-        wait(self.driver,15).until(EC.visibility_of_all_elements_located(all_options_loc))
+        all_options_loc = (
+            By.XPATH,'//a[@title="Originals"]/following-sibling::div//a')
+        wait(self.driver,15).until(
+            EC.visibility_of_all_elements_located(all_options_loc))
         all_options = self.driver.find_elements(*all_options_loc)
-        excluded_num = [2]   #list index of 2 is Baby Brain option title, this is not including as an option because it consistently changes its internal link's url.
-        random_option = all_options[Tools().generate_random_num(1,10,excluded_num)] 
+                                                                                #excluded_num = ['tab number'], an exclusion list was needed for past options because it consistently changed its internal link's url.
```

Comment:
Here I got creative in my comments to let anyone looking at my code know why I have excluded_num list which I'm not using.